starter-code/app.js

```diff
@@ -1,5 +1,43 @@
-const express = require('express');
+const express = require("express");
 const app = express();
-const Chuck  = require('chucknorris-io');
+const Chuck = require("chucknorris-io");
 const client = new Chuck();
 
+app.set("views", __dirname + "/views");
+app.set("view engine", "ejs");
+
+app.get("/random", (req, res, next) => {
+  // Retrieve a random chuck joke
+  client
+    .getRandomJoke()
+    .then(joke => {
+      res.render("index", joke);
+    })
+    .catch(err => {
+      console.log(err);
+      // handle error
+    });
+});
+
+app.get("/categories", (req, res, next) => {
+  client
+    .getJokeCategories()
+    .then(result => {
+      const data = {
```

Comment:
Here you are doing what the previous comment says, wrap the information you want to pass to the render in an objetc. Good job.